client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -48,7 +48,7 @@ class MobileLayout extends Component<MobileLayoutProps, MobileLayoutState> {
   };
 
   // Keep the tool panel visible when mobile address bar and/or keyboard are in view.
-  setToolPanelPosition = () => {
+  setToolPanelPosition = (): void => {
```

Comment:
some changes in typescript eslint packages started flagging this. which seems like a good thing right?